fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -4,7 +4,7 @@ grammar StarRocks;
 import StarRocksLex;
 
 sqlStatements
-    : (singleStatement (SEMICOLON EOF? | EOF))*
+    : (singleStatement (SEMICOLON EOF? | EOF))+
```

Comment:
What's the different?

Reply:
*A statement that does not meet the conditions will not report an error, because there is no match. + will report an error when there is no matching statement